ml-prims/test/CMakeLists.txt

```diff
@@ -28,8 +28,11 @@ add_executable(mlcommon_test
   cuda_utils.cu
   cov.cu
   decoupled_lookback.cu
-  distance.cu
-  # distance_adj.cu
+	dist_adj.cu
```

Comment:
knitpick! please adjust the indentation.

Reply:
Thanks. Both of them just fixed.